### ISSUE TITLE
"If it doesn't work for you, then don't use it" that's not how support works..?

### ISSUE THREAD

**Issue by DxvkOnGtaFiveeee** [doitsujin/dxvk#4044]
![image](https://github.com/doitsujin/dxvk/assets/171378640/a11d1fee-a11c-4e52-be5d-ee06a6c2c237)

What's the point of bug reports / issues and _support_ in general if that's the main "solution" ?
If my pc suddenly shows some error, I should just throw it away.
If running hurts my legs, I should just stop running.

Great logic my guy, great logic.

Sure, maybe you don't support Windows, but it seems like that would still be your answer if I used Wine, the latest DXVK, and a newer GPU.
What a pathetic excuse for support, honestly.

**Comment by Blisto91** [comment]
> Sure, maybe you don't support Windows, but it seems like that would still be your answer if I used Wine, the latest DXVK, and a newer GPU.

The short of it here is that we do not support old versions of dxvk. If you have issues with it on a old unsupported GPU on Windows then the best fix here would be not to use dxvk. Kepler is known to be iffy with Vulkan and even sometimes newer Direct3D versions.

**Comment by DxvkOnGtaFiveeee** [comment]
While I can see the logic behind not supporting old versions of DXVK, I believe supporting 1.10.3 specifically wouldn't be a bad thing. I'm saying that because 1.10.3 is the last version to support pre-1.3 Vulkan.

Using the latest version of DXVK is easy. Upgrading your GPU unfortunately isn't, and for the time being there are plenty of people with old GPUs.

The team behind DXVK doesn't have to shift all their focus to 1.10.3 obviously, but supporting it even a little instead of completely ignoring it wouldn't hurt anyone.

This doesn't apply to me, got the message. Just sayin'

**Comment by mbriar** [comment]
Even if anybody wanted there is nothing that could be done about improving performance in GTAV on ancient nvidia gpus without driver updates. DXVK exists to make games playable on linux, not to improve performance on windows. There is no need to use it on windows.

**Comment by K0bin** [comment]
> What's the point of bug reports / issues and support in general if that's the main "solution" ?
If my pc suddenly shows some error, I should just throw it away.
If running hurts my legs, I should just stop running.

Those errors are almost certainly caused by driver bugs. So go ahead and annoy Nvidia, not us. There's simply nothing we can do about it.

If you're on Windows, there is simply no reason to use DXVK anyway, especially on older hardware. Even if it was working perfectly, it would most likely just be slower than your regular Windows D3D11 driver.

I'd also like to remind you that you didn't pay anything for DXVK, so you also aren't *entitled* to get support.

**Comment by K0bin** [comment]
What exactly do you expect us to do?

Vulkan doesn't perform well on old Nvidia hardware and there's nothing we can do about that even if we wanted.
I'd also like to remind you that you didn't pay for DXVK, so you aren't *entitled* to get support.

> but supporting it even a little instead of completely ignoring it wouldn't hurt anyone.

It would waste our time.

**Comment by WinterSnowfall** [comment]
> Upgrading your GPU unfortunately isn't

Why not? We're no longer in the great GPU shortage that hit us a few years ago and there are plenty of last gen and previous gen GPUs still on the market, some with reasonable discounts. It's not terribly hard or even that expensive to get a modern low-end GPU with Vulkan 1.3 support that blows Kepler cards out of the water.

I doubt there are many projects out there than maintain legacy branches just in the interest of older hardware compatibility, that rarely makes much sense, neither in terms of time investment or economic value. And of course if anyone has a vested interest to maintain dxvk at 1.10.3 levels they are very much free to fork it and do so. That's the beauty of open source.

**Comment by mbriar** [comment]
But keep in mind that even with an upgraded GPU there still would be zero reason to use DXVK with GTAV on windows.

**Comment by DxvkOnGtaFiveeee** [comment]
> I'd also like to remind you that you didn't pay for DXVK, so you aren't _entitled_ to get support.
Github has the "Issues" feature, and the Discord server straight up has a "User Support" channel.
The DVXK team themselves **offer** support.

> It would waste our time.
That's an honest answer I can live with. 
When it comes to this, you _aren't forced to help_ and that's understandable. But if you DO happen to know a solution, then helping someone else is generally a good thing. In fact, you might be helping many people. 

That's how I see it, at least.

**Comment by DxvkOnGtaFiveeee** [comment]
> Why not?

I don't know what's your situation, but not everybody can afford some things just because _you_ can, as stupid as it might sound.

**Comment by DxvkOnGtaFiveeee** [comment]
> But keep in mind that even with an upgraded GPU there still would be zero reason to use DXVK with GTAV on windows.

Not true. Plenty of people have reported amazing FPS increases by using DXVK with GTAV on Windows.
It's not perfect overall, but it does work for some people.
